code/modules/mob/living/living_health_procs.dm

```diff
@@ -550,6 +550,8 @@
 		GLOB.alive_mob_list += src
 		tod = null
 		timeofdeath = 0
+	if(isanimal(src))
+		health = maxHealth
```

Comment:
i know we do this earlier in the proc, but typechecking src is kinda always wrong. can you override rejuvenate() for /mob/living/simple_animal and just set health = maxHealth before calling parent?